doc/source/serve/getting_started.md

```diff
@@ -2,9 +2,20 @@
 
 # Getting Started
 
-This tutorial will walk you through the process of using Ray Serve to deploy a single model behind HTTP locally.
+This tutorial will walk you through the process of deploying models with Ray Serve. It will show you how to
 
-We'll be using [HuggingFace's SummarizationPipeline](https://huggingface.co/docs/transformers/main_classes/pipelines#transformers.SummarizationPipeline) to deploy a model that summarizes text.
+* Expose your models over HTTP using Ray Serve `deployments`
+* Scale your deployments to meet your workload's requirements
+* Allocate resources like fractional GPUs and CPUs to your deployments
+* Compose multiple-model machine learning pipelines with Ray Serve `deployment graphs`
```

Comment:
Nit: monospace looks a little odd for `deployments` and `deployment graphs`.  Is there a deployment graph page we can link to here?

Reply:
I linked them both to their corresponding documentation instead of using monospace.